doc/source/development.rst

```diff
@@ -93,12 +93,12 @@ environment for Python 3:
    $ source .tox/3/bin/activate
 
 The commands above automatically creates the application script
-called :command:`kiwi-ng-3`, which allows you to run KIWI from the
+called :command:`kiwi`, which allows you to run KIWI from the
```

Comment:
shouldn't that be kiwi-ng  instead of just kiwi ?

Reply:
you are right